docs/src/main/sphinx/sql/show-functions.rst

```diff
@@ -12,10 +12,32 @@ Synopsis
 Description
 -----------
 
-List all the functions available for use in queries.
+List all the functions available for use in queries in a SQL client connected
+to Trino. For each function returned, the following information is displayed:
+
+* Function name
+* Return type
+* Argument types
+* Function type
+* Deterministic
+* Description
 
 :ref:`Specify a pattern <like_operator>` in the optional ``LIKE`` clause to
 filter the results to the desired subset. For example, the following query
 allows you to find functions beginning with ``array``::
 
     SHOW FUNCTIONS LIKE 'array%';
+
+``SHOW FUNCTIONS`` works with built-in functions as well as with :doc:`custom
+functions </develop/functions>`. In this example, three custom functions have
```

Comment:
In the following example three custom functions with the cf_ prefix are available: